src/smart-components/myUserAccess/MUAContent.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Stack, StackItem,
+  Title
+} from '@patternfly/react-core';
+
+import MUARolesTable from './MUARolesTable';
+import MUAAccessTable from './MUAAccessTable';
+import MUACard from '../../presentational-components/myUserAccess/MUACard';
+
+import './MUAContent.scss';
+
+const MUAContent = ({ entitlements, isOrgAdmin }) => {
+
+  const entitledBundles = Object.entries(entitlements).filter(([ , { is_entitled }]) => is_entitled);
```

Comment:
If you are going to use just value, you can do `Object.values` instead of `Object.entries`

Reply:
I think entries is wanted here because it's used in `MUACard`